index.js

```diff
@@ -1,10 +1,15 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
 */
+var loaderUtils = require('loader-utils');
+
 module.exports = function(source) {
-	this.cacheable && this.cacheable();
-	var value = typeof source === "string" ? JSON.parse(source) : source;
-	this.value = [value];
-	return "module.exports = " + JSON.stringify(value) + ";";
+    this.cacheable && this.cacheable();    
+    var value = typeof source === "string" ? JSON.parse(source) : source;
+    this.value = [value];
+    var query = loaderUtils.getOptions(this) || {}
+    if(query.string)
+	return "module.exports = `" + JSON.stringify(value) + "`;";
+    return "module.exports = " + JSON.stringify(value) + ";";
```

Comment:
Do we want to go webpack 2 here? Separate PR would be probably better for that change, though.